common/schema.h

```diff
@@ -500,11 +504,12 @@ namespace swss {
 #define DEL_COMMAND "DEL"
 #define EMPTY_PREFIX ""
 
-#define CFG_DTEL_TABLE_NAME					"DTEL"
-#define CFG_DTEL_REPORT_SESSION_TABLE_NAME	"DTEL_REPORT_SESSION"
-#define CFG_DTEL_INT_SESSION_TABLE_NAME		"DTEL_INT_SESSION"
-#define CFG_DTEL_QUEUE_REPORT_TABLE_NAME	"DTEL_QUEUE_REPORT"
-#define CFG_DTEL_EVENT_TABLE_NAME			"DTEL_EVENT"
+#define CFG_DTEL_TABLE_NAME                 "DTEL"
+#define CFG_DTEL_REPORT_SESSION_TABLE_NAME  "DTEL_REPORT_SESSION"
+#define CFG_DTEL_INT_SESSION_TABLE_NAME     "DTEL_INT_SESSION"
+#define CFG_DTEL_QUEUE_REPORT_TABLE_NAME    "DTEL_QUEUE_REPORT"
+#define CFG_DTEL_EVENT_TABLE_NAME           "DTEL_EVENT"
+
 
```

Comment:
schema.h file in master branch contains 393 lines on code. how come your file (which is based on master) contains more than 500 ?

Reply:
Not sure where you were looking at, but master has 500+ lines in src/sonic-swss-common/common/schema.h